tuftool/Cargo.toml

```
@@ -36,8 +36,8 @@ structopt = "0.3"
 tempfile = "3.3.0"
 tokio = "~1.18"  # LTS
 tough = { version = "0.12.2", path = "../tough", features = ["http"] }
```

```suggestion
tough = { version = "0.12.3", path = "../tough", features = ["http"] }
```
We should probably cut a new release of `tough` as well.